frontend/shared-components/error-form-level/error-form-level.tsx

```diff
@@ -0,0 +1,16 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons'
+import React from 'react'
+import styles from './error-form-level.module.css'
+
+interface FormLevelErrorProps { errorId: string, errorMessage: string }
```

Comment:
Can we use just type instead of Interface since that implys more generic reusability which we are not following that programming paradigm.